global-cli/index.js

```diff
@@ -166,3 +169,13 @@ function checkNodeVersion() {
     process.exit(1);
   }
 }
+
+function isGitHubBoilerplate(root) {
+  var validFiles = [
+    '.DS_Store', 'Thumbs.db', '.git', '.gitignore', 'README.md', 'LICENSE'
```

Comment:
Can you open a new issue with this request please? It's a little difficult to track in an issue like this.